File: mRemoteV1/Connection/Connection.Protocol.SSH2.cs


namespace mRemoteNG.Connection.Protocol
{
	public class SSH2 : PuttyBase
	{
				
		public SSH2()
		{
			this.PuttyProtocol = Putty_Protocol.ssh;
			this.PuttySSHVersion = Putty_SSHVersion.ssh2;
		}
				
		public enum Defaults
		{
			Port = 22
		}
	}
}

File: mRemoteV1/Connection/Connection.Protocol.SSH2.vb
Namespace Connection

    Namespace Protocol
        Public Class SSH2
            Inherits PuttyBase

            Public Sub New()
                Me.PuttyProtocol = Putty_Protocol.ssh
                Me.PuttySSHVersion = Putty_SSHVersion.ssh2
            End Sub

            Public Enum Defaults
                None = 0
                Port = 22
            End Enum
        End Class
    End Namespace

End Namespace